src/Console/Gameplay.cpp

```diff
@@ -147,7 +147,8 @@ void playerState() {
         } else {
             Serial << F("Gameplay: Player incorrect.  currentLength = ") << currentLength << endl;
             // if so, show the correct next button
-            play(correctSequence[correctLength], false);
+            //play(correctSequence[correctLength], false);
+            animateFailure();
```

Comment:
Consider calling fanFare(0) instead, and move the Light/Sound/Fire animations for failure (and levels of successes) to that code.